transcendence-app/src/chat/chat.controller.ts

```diff
@@ -187,6 +189,31 @@ export class ChatController {
     return messages;
   }
 
+  @Patch('room/:chatroomId')
+  @ApiBadRequestResponse({ description: 'Bad Request' })
+  @ApiUnprocessableEntityResponse({ description: 'Unprocessable Entity' })
+  async updateChatroom(
+    @GetUser() userMe: User,
+    @Param('chatroomId', ParseUUIDPipe) chatroomId: string,
+    @Body() updateChatroomDto: UpdateChatroomDto,
+  ): Promise<Chatroom> {
+    const chatroom = await this.getChatroomById(chatroomId);
+    if (!chatroom) {
+      throw new ServiceUnavailableException();
+    }
```

Comment:
question: What do you think about using the `NotFoundException` here? Because the `getById` method also returns null when there is no chatroom with that ID. `ServiceUnavailable` is also technically correct because the database server could be offline or malfunctioning.

suggestion: Please add an Api*Response decorator for the Swagger documentation.

Reply:
Ok, I'll do it

Reply:
Actually, the getById method thows the NotFound exception. So we don't need it here don't we?

Reply:
You are right :+1:
What do you think about moving the owner check in the `ChatService.addChatroomMember` method after finding the chatroom, and we could remove this part from here?

Reply:
Done